lib/voice/VoiceConnectionManager.js

```diff
@@ -83,6 +84,7 @@ class VoiceConnectionManager extends Collection {
             connection = this.add(new this.baseObject(data.guild_id, {
                 shard: data.shard,
                 opusOnly: this.pendingGuilds[data.guild_id].options.opusOnly ?? this.options.opusOnly,
+                udpTimeout: this.pendingGuilds[data.guild_id].options.udpTimeout ?? this.options.udpTimeout,
```

Comment:
Is this intentionally meant to be configurable from the `Client#joinVoiceChannel` call? If so, please also add a docstring there and in `VoiceChannel#join`